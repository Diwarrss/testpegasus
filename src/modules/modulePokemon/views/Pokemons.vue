<template>
  <div class="page-pokemones">
    <PokemonList />
    <SelectedPokemonTable />
  </div>
</template>
<script>
import PokemonList from "@/modules/modulePokemon/components/PokemonList.vue";
import SelectedPokemonTable from "@/modules/modulePokemon/components/SelectedPokemonTable.vue";
export default {
  components: {
    PokemonList,
    SelectedPokemonTable
  },
  created() {
    this.getPokemons()
  },
  methods: {
    getPokemons(){
      this.axios.get('https://pokeapi.co/api/v2/pokemon?limit=10')
      .then(res => {
        this.$store.commit('setPokemons', res.data)
      })
      .catch(err => {
        console.error(err); 
      })
    }
  }
}
</script>