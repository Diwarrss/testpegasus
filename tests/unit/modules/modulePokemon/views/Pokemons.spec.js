import { shallowMount } from "@vue/test-utils";
import Pokemons from "@/modules/modulePokemon/views/Pokemons";

describe("Pokemons.vue", () => {
  test("Verificar que el view no se modifico", () => {
    const wrapper = shallowMount(Pokemons);
    expect(wrapper.html()).toMatchSnapshot()
  });
});
