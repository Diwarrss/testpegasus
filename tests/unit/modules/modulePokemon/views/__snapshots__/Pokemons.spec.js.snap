// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemons.vue Verificar que el view no se modifico 1`] = `
<div class="page-pokemones">
  <pokemon-list-stub></pokemon-list-stub>
  <selected-pokemon-table-stub></selected-pokemon-table-stub>
</div>
`;
